### PR TITLE
add Online Timer

### PR DESCRIPTION
**Pull request by TowardtheStars**
---

## 检查以下事项 | Check the following:

- 仓库 URL、分支和 `related_path` 正确且 `mcdreforged.plugin.json` 位于应当存在的位置
  Repository URL, branch and `related_path` are correct and `mcdreforged.plugin.json` file exists at the correct path

- 根据 [文档](https://mcdreforged.readthedocs.io/zh_CN/latest/plugin_dev/plugin_catalogue.html#label)，检查插件标签正确且合适  
  Plugin labels are correct and appropriate according to [documentation](https://mcdreforged.readthedocs.io/en/latest/plugin_dev/plugin_catalogue.html#label)

- 确保 `mcdreforged.plugin.json` 和 `plugin_info.json` 中的插件 ID 与在本仓库创建的文件夹名称一致
  Plugin ID in `mcdreforged.plugin.json` and `plugin_info.json` match the name of the folder created in this repository

- 在介绍中简要阐明插件功能，在 README 中详细阐明插件用法，避免含糊不清。确保潜在用户有理由考虑使用您的插件
  Briefly clarify plugin features in the introduction, and clarify plugin usage in detail in the README. Avoid vague descriptions. Ensure that potential users have a reason to consider using your plugin

- [x] 我已检查并确认以上所有项目
  I have checked and confirmed all the above

### CODE REVIEW

**Comment by AnzhiZhang** [comment]
分类应该是 `management` 和 `information` 吧？`tool` 主要是计算器等工具

**Comment by TowardtheStars** [comment]
我看 Seen 这个插件写的也是 `tool`，然后插件的功能也比较类似，所以就把分类抄过来了。

File: plugins/online_timer/plugin_info.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "id": "online_timer",
+    "authors": [
+        {
+            "name": "TowardtheStars",
+            "link": "https://github.com/TowardtheStars"
+        }
+    ],
+    "repository": "https://github.com/TowardtheStars/MCDR-Online-Timer",
+    "branch": "main",
+    "labels": ["tool"],
```
Comment (AnzhiZhang) [inline plugins/online_timer/plugin_info.json:11]:
```suggestion
    "labels": ["information"],
```

**Comment by alex3236** [comment]
Closing as it's inactive through 2 months

**Pull request closed without merging**